metrics/grafana/tikv_details.json

```diff
@@ -14,7 +14,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "7.5.7"
+      "version": "7.5.11"
```

Comment:
How about only including necessary changes?

Reply:
it should be changed anyway, as the grafana has been updated

Reply:
Yes, but other changes like `gridPos` are not necessary.

Reply:
Does v5.0 use the same version?

Reply:
clean the unnecessary changes now